core/lib/types/src/eth_sender.rs

```diff
@@ -12,20 +12,6 @@ pub struct EthTx {
     pub predicted_gas_cost: u64,
 }
 
-impl std::fmt::Debug for EthTx {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        // Do not print raw_tx
-        f.debug_struct("EthTx")
-            .field("id", &self.id)
-            .field("nonce", &self.nonce)
-            .field("contract_address", &self.contract_address)
```

Comment:
why so? do we really need raw_tx bytes? it could be 128kb long

Reply:
@Deniallugo It is just rolling back to how it is done in the `main` branch. https://github.com/matter-labs/zksync-era/pull/112#discussion_r1377461695

Reply:
Ok!

Reply:
I'd rather not have raw tx bytes in the output, main reason why I implemented the custom Debug function for this. They are not helpful for debugging at all, and simply take space in the terminal and obstruct actually useful logs.

Reply:
ok